Lib/test/test_unparse.py

```diff
@@ -705,7 +705,7 @@ class DirectoryTestCase(ASTTestCase):
     test_directories = (lib_dir, lib_dir / "test")
     run_always_files = {"test_grammar.py", "test_syntax.py", "test_compile.py",
                         "test_ast.py", "test_asdl_parser.py", "test_fstring.py",
-                        "test_patma.py"}
+                        "test_patma.py", "test_type_alias.py"}
```

Comment:
```suggestion
                        "test_patma.py", "test_type_alias.py", "test_type_params.py"}
```
Maybe this one too? It tests a lot of PEP 695 syntax.